sycl/include/CL/sycl/detail/aligned_allocator.hpp

```diff
@@ -8,7 +8,7 @@
 
 #pragma once
 
-#include <CL/cl.h>
+#include <CL/sycl/detail/cl.h>
```

Comment:
I would just remove `#include <CL/cl.h>` for two reasons:
1. I don't see where declarations from this headers are used here.
2. Even if I missed something, next line includes `common.hpp`, which should cover us.

Reply:
Will do.